src/main/java/core/gui/theme/GroupTeam.java

```diff
@@ -0,0 +1,31 @@
+package core.gui.theme;
+
+import com.kitfox.svg.app.beans.SVGIcon;
+
+public class GroupTeam {
```

Comment:
I am wondering if separating this from `ThemeManager` will prevent us from selecting different icons based on the theme?

Also `ThemeManager` handles caching of the icons, which is important: I forgot to cache the jersey SVG icons, and the performance hit was quite significant.

Reply:
see my last comment, but yes we can remove this class and it in ThemeManager